src/vtk/ObliqueRepresentationProxy/index.js

```diff
@@ -0,0 +1,121 @@
+import { vec3 } from 'gl-matrix';
+
+import macro from '@kitware/vtk.js/macro';
+import vtkActor from '@kitware/vtk.js/Rendering/Core/Actor';
+import vtkCubeSource from '@kitware/vtk.js/Filters/Sources/CubeSource';
+import vtkCutter from '@kitware/vtk.js/Filters/Core/Cutter';
+import vtkMapper from '@kitware/vtk.js/Rendering/Core/Mapper';
+import vtkPolyData from '@kitware/vtk.js/Common/DataModel/PolyData';
+import vtkResliceRepresentationProxy from '@kitware/vtk.js/Proxy/Representations/ResliceRepresentationProxy';
+
+function vtkObliqueRepresentationProxy(publicAPI, model) {
+  model.classHierarchy.push('vtkObliqueRepresentationProxy');
+
+  function transformPoints(points, transform) {
+    const tmp = [0, 0, 0];
+    for (let i = 0; i < points.length; i += 3) {
+      const p = points.subarray(i, i + 3);
+      if (transform.length === 9) {
+        vec3.transformMat3(tmp, p, transform);
+      } else {
+        vec3.transformMat4(tmp, p, transform);
+      }
+      [p[0], p[1], p[2]] = tmp;
+    }
```

Comment:
`vtkMatrixBuilder.apply` does that already

Reply:
This is refactored into vtk-js here: https://github.com/Kitware/vtk-js/pull/2908
Will be removing ObliqueRepresentationProxy from Volview.